src/beamlime/applications/_nexus_helpers.py

```diff
@@ -222,7 +264,10 @@ def collect_streaming_modules(
     )
```

Comment:
Why first create a tuple of pairs and then make it into a dict?
Why not just make the dict directly?

Reply:
Aha okay 👍